node_modules/vue-json-compare/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
Why do we have node_modules folder? Can we remove it and reference required scripts via script tag?